packages/store/src/decorators/select/symbols.ts

```diff
@@ -46,8 +50,36 @@ export function removeDollarAtTheEnd(name: string): string {
   return dollarAtTheEnd ? name.slice(0, lastCharIndex) : name;
 }
 
+export type SelectorFn =
+  | ((state: any, ...states: any[]) => any)
+  | string
+  | Type<any>
+  | StateToken<any>;
+
 export type PropertyType<T> = T extends StateToken<any>
   ? Observable<ExtractTokenType<T>>
   : T extends (...args: any[]) => any
   ? Observable<ReturnType<T>>
   : any;
+
+function createSelectObservableIvy<T = any>(
+  selector: any,
+  store: Store | null
+): Observable<T> {
+  if (ngDevMode && !store) throwSelectFactoryNotConnectedError();
```

Comment:
more readable


```ts
 if (ngDevMode && !store) { 
    throwSelectFactoryNotConnectedError();
 }
```

Reply:
I agree that this is more readable, but I don't mind either way.
I think that the only time that you could potentially leave out the block is when it is a guard clause or assertion (like this). But, maybe it is just better to always do the block for consistency.
I think prettier actually has a rule to force this if we choose to make a general rule.